### PR TITLE
[FEAT]: rocket animation for doge

### PR DESCRIPTION
**Pull request by rastajpa**
rocket

### CODE REVIEW

File: src/pages/finish/finish.ts [hunk 1]
```diff
@@ -32,9 +113,23 @@ export class FinishModalPage {
         this.viewCtrl.dismiss();
       }, 4000);
     }
+    if (this.navParams.data.coin === 'doge' && this.cssClass === 'success') {
```
Comment (cmgustavo) [inline src/pages/finish/finish.ts:116]: I'd add a control to check if `coin` is undefined. What do you think?

**Review by cmgustavo — DISMISSED** [review]
LGTM! Just add a control to check if `coin` param doesn't exist

File: src/pages/finish/finish.ts (hunk 1 above)
Comment (rastajpa) [reply src/pages/finish/finish.ts:116]: yes, that make sense...

**Comment by rastajpa** [comment]
> LGTM! Just add a control to check if `coin` param doesn't exist

Done, ty!

**Review by cmgustavo — APPROVED** [review]
ACK 🚀

**Pull request merged**